client/stylesheets/_toCat.scss

```diff
@@ -4582,10 +4583,16 @@ On click the images will be displayed at normal size to complete the effect
 .searchResultsRefine ul li{
     font-size: 0.9rem;
     margin: 5px 0;
+    margin-left: 10px;
+    line-height: 1.5;
 }
 
 .searchResultsRefine ul > p {
-    margin-bottom: 0px;
+    margin-bottom: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, color, margin-bottom, margin-top, padding